mock-up/mock.ts

```diff
@@ -89,13 +89,23 @@
             "subTitle": "AWS Billing event triggers a Slack message",
             "id": '20230406_1'
 },
+
 {
             sinkIcon: "discord",
             source: 'amazon-eventbridge',
             sink: 'http',
             "description": "Receive real-time notifications on Discord when someone uses Amazon S3 to create or delete Bucket.",
             "subTitle": "Amazon EventBridge CreateBucket | DeleteBucket event triggers a Discord message",
             "id": '20230406_2'
+},
+
+{
+            sinkIcon: "slack",
+            source: 'amazon-eventbridge',
+            sink: 'http',
+            "description": "Receive real-time notifications on a Slack Channel when someone uses Amazon S3 to create or delete Bucket.",
+            "subTitle": "Amazon EventBridge CreateBucket | DeleteBucket event triggers a Slack message",
+            "id": '20230406_2'
```

Comment:
ID is not consistent with that of the template file
Kindly use current date as ID